CMakeLists.txt

```diff
@@ -58,6 +58,9 @@ if(EVMC_TESTING OR EVMC_TEST_TOOLS)
 endif()
 
 if(EVMC_TESTING OR EVMC_EXAMPLES)
+    hunter_add_package(ethash)
```

Comment:
Had this in `examples/CMakeLists.txt` first but hoped this would fix the linking issue. Too bad there doesn't seem to be a way to statically link with ethash via `target_link_libraries`.

@chfast any simple solutions?

Reply:
You need ethash 0.5.0: https://github.com/chfast/ethash/blob/master/CHANGELOG.md#050---2019-06-07.

Reply:
Doesn't hunter have 0.5.0 ?

Reply:
Ah, hunter has 0.5.0, but evmc still uses an old hunter.